lib/school.rb

```diff
@@ -0,0 +1,43 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(student)
+    @student_names << student
+  end
+
+  def end_time
+    converted_start_time = @start_time.to_i
+    calculated_time = converted_start_time + @hours_in_school_day
+    end_time = calculated_time.to_s + ":00"
+    return end_time
```

Comment:
This works, but you do not need line 20, or the variable assignment on line 19.  You could end this method with `calculated_time.to_s + ":00"`